src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx

```diff
@@ -31,32 +47,67 @@ export interface Address {
   postalCode: string
 }
 
-interface ConformBidState {
+interface StripeToken {
+  tokenId: string
+  created: number
+  livemode: 1 | 0
```

Comment:
What is this/why do we need to pass it?

Reply:
This is the token that comes through the tipsi-stripe library. `tokenId` is the string token, and the other bits come along with it. The individual poperties are in the docs [here](https://tipsi.github.io/tipsi-stripe/docs/token.html)